float64 v_x
float64 v_y
float64 v_th
---
float64[] robot_state
float32[] lidar_measurements
float64 reward
bool done